jsonschema-core.xml

```diff
@@ -1563,206 +1587,84 @@
-                        </t>
-                        <figure>
-                            <preamble>
-                                Consider the following two schemas.  The first schema, identified
-                                as "original" as it is the schema to be extended, describes
-                                an object with one string property and one recursive reference
-                                property, "r".  The second schema, identified as "extension",
-                                references the first, and describes an additional "things" property,
-                                which is an array of recursive references.
-                                It also repeats the description of "r" from the original schema.
-                            </preamble>
-                            <artwork>
-<![CDATA[
-{
-    "$schema": "https://json-schema.org/draft/2019-08/schema",
-    "$id": "https://example.com/original",
-
-    "properties": {
-        "name": {
-            "type": "string"
-        },
-        "r": {
-            "$ref": "#"
-        }
-    }
-}
-
-{
-    "$schema": "https://json-schema.org/draft/2019-08/schema",
-    "$id": "https://example.com/extension",
-
-    "$ref": "original",
-    "properties": {
-        "r": {
-            "$ref": "#"
-        },
-        "things": {
-            "type": "array"
-            "items": {
-                "$ref": "#"
-            }
-        }
-    }
-}
-]]>
-                            </artwork>
-                            <postamble>
-                                This apparent duplication is important because
-                                it resolves to "https://example.com/extension#", meaning that
-                                for instance validated against the extension schema, the value
-                                of "r" must be valid according to the extension, and not just the
-                                original schema as "r" was described there.
-                            </postamble>
-                        </figure>
-                        <t>
-                            This approach is fine for a single recursive field, but the more
-                            complicated the original schema, the more redefinitions are necessary
-                            in the extension.  This leads to a verbose and error-prone extension,
-                            which must be kept synchronized with the original schema if the
-                            original changes its recursive fields.
-                            This approach can be seen in the meta-schema for JSON Hyper-Schema
-                            in all prior drafts.
+                            For an example using these keyword, see appendix
+                            <xref target="recursive-example" format="counter" />.
+                            <cref>
+                                The difference between the hyper-schema meta-schema in previous
```

Comment:
Should we be talking about hyper-schema in core?

Reply:
@philsturgeon it's in a CREF.  It's not really part of the spec, it's just commentary and it's by far the clearest example of why this feature is needed.  The old hyper-schema meta-schema is unmaintainable.  I broke it a lot trying to keep it in sync.

Also, this hyper-schema reference is not new, so if you want to take it out, that's a separate issue.  I just moved it and maybe reworded it.